provision/filter_plugins/aws_inv.py

```diff
@@ -8,7 +8,9 @@
     from configparser import ConfigParser
 except ImportError:
     from ConfigParser import ConfigParser
-sys.path.append(os.getcwd())
+filepath = os.path.realpath(__file__)
```

Comment:
Two separate packages is fine with me. something like linchpin-modules and linchpin-filters seems appropriate.